Source/Noesis.Javascript/JavascriptInterop.cpp

```diff
@@ -710,7 +723,7 @@ JavascriptInterop::Invoker(const v8::FunctionCallbackInfo<Value>& iArgs)
             // not detect where nulls have been supplied (or insufficient parameters
             // have been supplied), but the corresponding parameter cannot accept
             // a null.  This will trigger an exception during invocation.
-			if (iArgs.Length() <= parametersInfo->Length)
+			if (suppliedArguments->Length <= parametersInfo->Length)

```

Comment:
An alternative to counting the maximum number of parameters upfront would be to add the following check here instead:
```cs
int parameterCount = suppliedArguments->Length;
if (i == members->Length - 1 && parameterCount > parametersInfo->Length)
    parameterCount = parametersInfo->Length; // cap parameter count if we reached the last available method and we have too many parameters
if (parameterCount <= parametersInfo->Length)
...
```
This saves us from doing the extra iteration over the method list but assumes that the method list is sorted by parameter count which I don't like that much. What do you think?

Reply:
I think it's fine.